todo-app/src/state.js

```diff
@@ -1,11 +1,36 @@
 import React from 'react'
 
-export default class State extends React.Component{
+export default class Status extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+          done: this.props.status
+        }
+        this.changedStatus = this.changedStatus.bind(this);
+      }
+      changedStatus(){
+          let changed = !this.state.done
+          this.setState({
+              done: changed
+         })
+
+        }
+      getStatus() {
+          let undone = "Undone"
```

Comment:
`undone` here is not the best name for this variable as it can get both `undone` and `done values.